tool/tsh/tsh.go

```diff
@@ -972,7 +969,7 @@ func onBenchmark(cf *CLIConf) {
 	fmt.Printf("\n")
 
 	if cf.BenchExport {
-		path, err := benchmark.ExportLatencyProfile(cf.BenchExportPath, result.Histogram, cf.BenchTicks, cf.BenchValueScale)
+		path, err := benchmark.ExportLatencyProfile(cf.BenchExportPath, result.ServiceHistogram, cf.BenchTicks, cf.BenchValueScale)
```

Comment:
Do we want the response histogram here?  Admittedly, with the coordinated omission fixed, they should be more or less the same.

Better yet, why not export both -- as the plotter can accept different files.  That way users can do an eyeball check to make sure that response & service look similar.

Reply:
I think exporting both is a good idea.